app/views/orga/seasons/show.html.slim

```diff
@@ -10,6 +10,12 @@ h1 Season #{@season.name}
     dd = @season.starts_at
     dt End Date
     dd = @season.ends_at
+    dt Applications open at
+    dd = @season.applications_open_at
+    dt Applications close at
+    dd = @season.applications_close_at
+
+    
```

Comment:
There is also `project_proposals_open_at` and `project_proposals_close_at`. Could you please add those, too? Thank you!